protostar/commands/calculate_account_address_command.py

```diff
@@ -0,0 +1,65 @@
+from argparse import Namespace
+from logging import Logger
+from typing import Optional
+
+from protostar.cli import ProtostarCommand, MessengerFactory
+from protostar.io import LogColorProvider, StructuredMessage
+from protostar.starknet.account_address import AccountAddress
+from protostar.cli.common_arguments import (
+    ACCOUNT_CLASS_HASH_ARG,
+    ACCOUNT_ADDRESS_SALT_ARG,
+    ACCOUNT_CONSTRUCTOR_INPUT,
+)
+
+
+class AccountAddressMessage(StructuredMessage):
+    def __init__(self, account_address: AccountAddress) -> None:
+        super().__init__()
+        self._account_address = account_address
+
+    def format_human(self, fmt: LogColorProvider) -> str:
+        return fmt.colorize("CYAN", str(self._account_address))
+
+    def format_dict(self) -> dict:
+        return {"account_address": str(self._account_address)}
```

Comment:
```suggestion
        return {"address": str(self._account_address)}
```

I think it's redundant?